src/EloquentHashids.php

```diff
@@ -51,4 +51,12 @@ public static function getHashidEncodingValue(Model $model)
   {
     return $model->getKey();
   }
+  
+  /**
+   * @return string
+   */
+  public function getRouteKeyName()
+  {
+      return 'uid';
```

Comment:
Although, I just remembered.... I don't know if there is a way to access the current model. The `getHashidColumn` method expects it as parameter.